api_items.py

```diff
@@ -0,0 +1,65 @@
+from Item import Item
+from Status import Status
+import requests
```

Comment:
You need to add requests to requirements.txt

Reply:
Ran `pip freeze > requirements.txt` and pushed in [7e28c8a](https://github.com/rysharprules/DevOps-Course-Starter/pull/2/commits/7e28c8a1767f76bb8d9b6ca560c7d6ce52e6b78e)